radicle-cli/Cargo.toml

```diff
@@ -18,6 +18,7 @@ json-color = { version = "0.7" }
 lexopt = { version = "0.2" }
 log = { version = "0.4", features = ["std"] }
 nonempty = { version = "0.8" }
+regex = { version = "1.7" }
```

Comment:
:-1:

Reply:
Regex is a big crate -- not worth depending on just for a test.

Reply:
The pattern matching has been re-implemented in Rust with good testing https://github.com/radicle-dev/heartwood/pull/518/commits/12bbca9890fb3a855e7b837b43f7bc381018c1e8